core/common/src/thrift/file_system_master.thrift

```diff
@@ -161,13 +165,14 @@ service FileSystemMasterClientService extends common.AlluxioService {
    * If the path points to a directory, the method returns a list with file information for the
    * directory contents.
    */
-  list<FileInfo> listStatus( /** the path of the file or directory */ 1: string path)
+  list<FileInfo> listStatus( /** the path of the file or directory */ 1: string path,
+      /** listStatus options */ 2: ListStatusTOptions options)
     throws (1: exception.AlluxioTException e)
 
   /**
    * Loads metadata for the object identified by the given Alluxio path from UFS into Alluxio.
+   * This is deprecated. Use listStatus.
```

Comment:
for consistency with other methods in this file do:

```
/**
 * Loads ...
 *
 * THIS METHOD IS DEPRECATED SINCE VERSION 1.1 AND WILL BE REMOVED IN VERSION 2.0.
*/
```